lib/api/core/entrypoints/embedded/protocols/http.js

```diff
@@ -74,30 +73,32 @@ class HttpProtocol extends Protocol {
     this.decoders = this._setDecoders();
 
     this.server.on('request', (request, response) => {
+      const ips = [request.socket.remoteAddress];
+
+      if (request.headers['x-forwarded-for']) {
+        request.headers['x-forwarded-for'].split(',').forEach(s => ips.push(s.trim()));
+      }
+
       const
+        connection = new ClientConnection('HTTP/' + request.httpVersion, ips, request.headers),
         payload = {
-          requestId: uuid.v4(),
+          ips,
+          requestId: connection.id,
           url: request.url,
           method: request.method,
           headers: request.headers,
-          content: ''
-        },
-        ips = [request.socket.remoteAddress];
-      let stream;
-
-      if (request.headers['x-forwarded-for']) {
-        request.headers['x-forwarded-for'].split(',').forEach(s => ips.push(s.trim()));
-      }
+          content: '',
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this trailing comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComma)